test/Abp.ZeroCore.Tests/Zero/Users/UserManager_Tokens_Tests.cs

```diff
@@ -31,6 +31,8 @@ public async Task Should_Valid_Non_Expired_TokenValidityKey()
                 var user = await _abpUserManager.GetUserByIdAsync(AbpSession.GetUserId());
                 var tokenValidityKey = Guid.NewGuid().ToString();
                 await _abpUserManager.AddTokenValidityKeyAsync(user, tokenValidityKey, DateTime.UtcNow.AddDays(1));
+                await _unitOfWorkManager.Current.SaveChangesAsync();
```

Comment:
This seems unintuitive (and silently breaking behaviour at compile time, unlike constructor changes).

I think we should either add `[UnitOfWork]` or call `SaveChangesAsync` in `AddTokenValidityKeyAsync` if we use the repository.

Reply:
`SaveChanges `is rarely called in the methods of store and manager.
In order not to change this convention, I said it has some breaking changes.

Reply:
That is not true.

[`AbpUserStore`](https://github.com/aspnetboilerplate/aspnetboilerplate/blob/v5.1.0/src/Abp.ZeroCore/Authorization/Users/AbpUserStore.cs), which you modified to use the repository in `AddTokenValidityKeyAsync`, has several calls to `SaveChanges`.